monitoring/update-bucket-capacity-info-cronjob/alerts.yaml

```diff
@@ -0,0 +1,99 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: update_bucket_capacity_info_cronjob
+    type: constant
+    value: artesca-data-ops-update-bucket-capacity-info
+  - name: update_bucket_capacity_info_job_duration_threshold
+    type: config
+    value: 240
+  - name: update_bucket_capacity_info_success_job_existence_duration_threshold
+    type: config
+    value: 600
```

Comment:
if no successful jobs in 10m which means bucket capacityInfo meta is not being updated since 10m, so Veaam will refuse to interact with Artesca buckets if the data is longer than 10m.